validation/src/main/scala/hmda/validation/dsl/PredicateCommon.scala

```diff
@@ -54,7 +54,8 @@ object PredicateCommon {
   def numeric[T]: Predicate[T] = new Predicate[T] {
     override def validate: (T) => Boolean = _.asInstanceOf[AnyRef] match {
       case n: Number => true
-      case _ => throw new NotImplementedError("'numeric' doesn't handle string (or other) values yet")
+      case s: String => Try(s.toDouble).isSuccess
```

Comment:
Maybe save this branch for later, in case it's needed? I don't think we should introduce code that is not going to be used, it will linger in the codebase without being properly tested.